rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -236,13 +237,100 @@
         }
       }
     },
-    "is_part_of": {
-      "title": "Is part of",
-      "description": "Title of the host document.",
-      "type": "string",
-      "minLength": 1,
-      "form": {
-        "hide": true
+    "partOf": {
+      "title": "Part of (host document)",
+      "description": "Host document, for example a journal or a series (MARC 773, 800, 830).",
```

Comment:
Do we have to say it comes from MARC?

Reply:
@rerowep Yes, this description comes from the metadata group specifications. It's there to help librarian, at least the ones that can speak MARC fluently.

You'll have to read the specification files… :smile:

Reply:
Yes